coka/doctrine-encrypt-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Oka\\Doctrine\\EncryptBundle\\OkaDoctrineEncryptBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "OKA_DOCTRINE_ENCRYPT_SECRET": "%generate(secret)%%generate(secret)%"
```

Comment:
Why do you need two secrets? 
I think this would be okey with just one `%generate(secret)%`

Reply:
HI

In the doc I read that the function generate random alphanumeric sequence of 16 bytes I wanted to have some on 32 bytes